hack/make-rules/test-cmd-util.sh

```diff
@@ -3107,6 +3107,23 @@ run_daemonset_tests() {
   set +o errexit
 }
 
+run_rollout_tests() {
+  set -o nounset
+  set -o errexit
+
+  ### Show the status of the rollout
+  # create namespace 'my-namespace'
```

Comment:
Here we usually use `create_and_use_new_namespace` instead.

Reply:
i want test rollout status will recoginze namespace in my local yaml file. and my yaml have namespace `NAME`